manifest.json

```diff
@@ -17,12 +17,12 @@
         "url": "datamol.org"
     },
     "requirements": {
-        "yunohost": ">= 2.7.2"
+        "yunohost": ">= 3.5"
     },
     "multi_instance": true,
     "services": [
         "nginx",
-        "php7.2-fpm",
+        "php7.0-fpm",
```

Comment:
why is it 7.0 ? shouldn't it be 7.3?

Reply:
It's because php7.0-fpm is a native service provided by YunoHost as we are actually on Stretch. Will be php7.3-fpm when YunoHost 4.0 Buster will be ready. Work is ongoing ^^

Reply:
It looks wrong, because the `.sock` files mention 7.3. Just want to point out that 7.0 is not supported by Kimai. 
But it seems that you know what you are talking about and have a plan!

Reply:
Yes I know that it can seems wrong but PHP7.3 is installed by the package kimai2_ynh, it's not, for now, a native YunoHost service ^^

That section of the manifest is to declare the YunoHost services needed the application as described in https://yunohost.org/#/packaging_apps_manifest_en

For Stretch, php7.0-fpm should be removed from the list as we don't need php7.0-fpm.
And for future buster, it should be php7.3-fpm, but if we write that, there will be an issue during installation on Stretch.
When YunoHost 4.0 Buster will be sent out, it will manage that php stuff...

Reply:
Resolved then 😄

Reply:
And thanks for the explanation!